src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -44,6 +48,52 @@ private ScannedDocumentsHelper() {
             .collect(toList());
     }
 
+    public static void setExceptionRecordIdToScannedDocuments(
+        ExceptionRecord exceptionRecord,
+        CaseUpdateDetails caseDetails
+    ) {
+        List<String> dcns = exceptionRecord.scannedDocuments == null
+            ? emptyList()
+            : exceptionRecord.scannedDocuments
+            .stream()
+            .map(scannedDocument -> scannedDocument.controlNumber)
+            .collect(toList());
+        @SuppressWarnings("unchecked")
```

Comment:
missing extra empty line here? :monocle_face:

Reply:
still missing. not a blocker. just thought it would be nice to have spare line before annotation. btw, i'm surprised it worked. wouldn't be if paid attention more but one doesn't have. still - nice to know